src/lib/react/jsx/JsxStaticMacro.hx

```diff
@@ -22,10 +22,36 @@ private enum MetaValueType {
 class JsxStaticMacro
 {
 	static public var META_NAME = ':jsxStatic';
+	static public var FIELD_NAME = '__jsxStatic';
 
 	static var hasBeenHooked = false;
 	static var decls:Array<JsxStaticDecl> = [];
 
+	static public function build():Array<Field>
+	{
+		var fields = Context.getBuildFields();
```

Comment:
We can avoid calling `getBuildFields` and return `null` if we don't need to modify the fields. What would Tink do? ;)

Reply:
I always forget about this :)

Reply:
And now the cost is almost null, it takes 0.015s on my 1.4s build.